roles/common/tasks/nagios.yml

```diff
@@ -82,15 +82,18 @@
   register: firewalld
   ignore_errors: true
   no_log: true
-  when: ansible_os_family == "RedHat"
+  when: ansible_os_family == "RedHat" and ansible_distribution_major_version|int >= "7"
```

Comment:
When you do `and ansible_distribution_major_version|int`, the comparison (`7` in this case) shouldn't be double-quoted because it's comparing a number with a string.